common/expirecontainer.h

```diff
@@ -372,3 +373,124 @@ class ObjectCache : public ObjectCacheBase {
         return borrow(key, [] { return new ValEntity(); });
     }
 };
+
+template <typename KeyType, typename NodeType>
+class ObjectCache<KeyType, intrusive_list<NodeType>> : public ObjectCacheBase {
```

Comment:
inherit ObjectCache<KeyType, ValPtr>, so that Borrow can be reused.

Reply:
Borrow in list is not a implemention.
Those override operators are different, so as `Item` type. Inheritance may not help.